src/main/java/com/nha/abdm/wrapper/hiu/hrp/dataTransfer/requests/HIUClientHealthInformationRequest.java

```diff
@@ -13,7 +13,4 @@
 public class HIUClientHealthInformationRequest {
   private String requestId;
   private String consentId;
-  private String fromDate;
```

Comment:
lets make these changes in `hiu-facade.yaml` and `HIUController.java` as well

Reply:
Modified the changes.